package.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+    <name>rj_robocup</name>
+    <version>0.0.0</version>
+    <description>TODO: Package description</description>
+    <maintainer email="kylestach99@gmail.com">Kyle Stachowicz</maintainer>
+    <maintainer email="oswinso@gmail.com">Oswin So</maintainer>
+    <license>TODO: License declaration</license>
```

Comment:
We should probably agree on a license. It looks like we're on Apache right now: https://github.com/RoboJackets/robocup-software/blob/staging/LICENSE. I don't have any major concerns with carrying it over, but we should make sure everyone's still okay with that.

Reply:
Yeah, but we should just do that sometime later.